CLI/clitree/cli-xml/sonic-feature.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (C) 2023 NIPPON TELEGRAPH AND TELEPHONE CORPORATION.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<CLISH_MODULE
+  xmlns="http://www.dellemc.com/sonic/XMLSchema"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xsi:schemaLocation="http://www.dellemc.com/sonic/XMLSchema
+  http://www.dellemc.com/sonic/XMLSchema/clish.xsd"
+  >
+  <!--=========================================================================-->
+
+  <VIEW name="configure-view">
+
+    <COMMAND name="feature" help="Configure additional feature"></COMMAND>
+    <COMMAND name="feature next-hop-group" help="Next-hop Groups feature"></COMMAND>
+    <COMMAND name="feature next-hop-group enable" help="Enable Next-hop Groups feature">
+      <ACTION>
+        python3 $SONIC_CLI_ROOT/sonic-cli-feature.py configure_sonic_nexthop_groups 1
```

Comment:
In the HLD it is mentioned that the enable disable cannot be changed in run time.  Then why do we have a CLI which can modify it? If it doesn't take effect we should have a log message asking the user to save and reboot